apps/web/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@axon/admin-panel",
-  "version": "4.1.0",
+  "version": "4.2.0",
```

Biome (2.4.15)

[error] The key version was already declared.

(lint/suspicious/noDuplicateObjectKeys)